dispatch/static/manager/src/js/actions/UserActions.js

```diff
@@ -46,3 +46,20 @@ export function authenticateUser(email, password, nextPath = '/') {
       })
   }
 }
+
+export function deleteArticles(token, articleIds) {
+  return {
+    type: types.DELETE_ARTICLES,
+    payload: DispatchAPI.articles.deleteArticles(token, articleIds)
+      .then( json => json.deleted )
+  }
+}
+
+export function unauthenticateUser(token) {
+  return {
+    type: types.AUTH_DELETE_TOKEN,
+    payload: DispatchAPI.auth.deleteToken(token)
+      .then( json => json )
```

Comment:
it is the identity function, what does it accomplish? it sparked a mildly interesting back and forth. 👍 taking it out since it's done its job.